azurerm/internal/services/web/tests/resource_arm_app_service_test.go

```diff
@@ -3041,6 +3043,9 @@ resource "azurerm_app_service" "test" {
 
   logs {
     application_logs {
+      file_system {
+        level = "Warning"
+      }
```

Comment:
as there doesn't seem to be any other configuration for file system would this be better as
```suggestion
      file_system_level = "Warning"
```